usr/src/cmd/gui-install/src/welcome-screen.c

```diff
@@ -85,7 +127,11 @@ on_releasenotesbutton_clicked(GtkWidget *widget,
 				}
 			}
 		}
-		execl(XDG_OPEN, XDG_OPEN, RELEASENOTESURL, (char *)0);
+		if (extract_version(version, MAXVERLEN)) {
+			snprintf(url, MAXDOCURLSIZE, RELEASENOTESURLFORMAT, version);
+			execl(XDG_OPEN, XDG_OPEN, url, (char *)0);
+		}
```

Comment:
We should probably have some reasonable default, if something goes wrong, e.g. https://docs.openindiana.org or https://docs.openindiana.org/release-notes/latest-changes/, if we put there link to latest release notes, e.g. 

2019-11-05 OpenIndiana 2019.10 released, see the [release notes](https://docs.openindiana.org/release-notes/2019.10-release-notes/)